docs/docs/policies.mdx

```diff
@@ -396,6 +398,268 @@ for details.
+|                                       |                        | logged. Either after every epoch ('epoch') or for every      |
+|                                       |                        | training step ('batch').                                     |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| checkpoint_model                      | False                  | Save the best performing model during training. Models are   |
+|                                       |                        | stored to the location specified by `--out`. Only the one    |
+|                                       |                        | best model will be saved.                                    |
+|                                       |                        | Requires `evaluate_on_number_of_examples > 0` and            |
+|                                       |                        | `evaluate_every_number_of_epochs > 0`                        |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| featurizers                           | []                     | List of featurizer names (alias names). Only features        |
+|                                       |                        | coming from the listed names are used. If list is empty      |
+|                                       |                        | all available features are used.                             |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| ignore_intents_list                   | []                     |                                                              |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| tolerance                             | 0.0                    |                                                              |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+```
+
+</details>
+
+#### Tuning the tolerance parameter
+
+Users are also encouraged to tune the `tolerance` parameter in `IntentTEDPolicy`'s configuration to reduce the number
+of false warnings. As you increase the value of `tolerance` from `0` to `1` in steps of `0.05`,
+the number of false warnings should decrease. However, increasing the `tolerance` will
+also result in fewer triggers of `action_unlikely_intent` and hence more number of conversation
+paths not present in training stories missing from the set of flagged conversations. This is particularly helpful
+when you are [reviewing real conversations](./conversation-driven-development.mdx#review) that
+your assistant has processed.
```

Comment:
I am not sure what parts of it we can add here. Are those scripts shareable?
Wdyt @kedz

Reply:
I don't think it would be trivial to share those scripts at this point. 
Beyond cleaning up/better documenting the code, there are some pretty significant hacks I had to do:
1. Make an environment variable that adjusted the tolerance parameter at test time
2. Disable the check to not return an action_unlikely_intent if the user intent was the most likely intent predicted by IntentTED (even though its score was below the threshold).
3. The scripts assume that `utter_unlikely_intent` warnings are in the `failed_test_stories.yml` and would need to be changed to look in the warnings file.

The third issue is trivial, but 1 and 2 seem more complicated to change, but perhaps we can discuss today?